primitives/api/proc-macro/src/impl_runtime_apis.rs

```diff
@@ -163,7 +163,7 @@ fn generate_impl_calls(
 				)?;
 
 				impl_calls.push(
-					(impl_trait_ident.clone(), method.sig.ident.clone(), impl_call)
+					(impl_trait_ident.clone(), method.sig.ident.clone(), impl_call, impl_.attrs.clone())
```

Comment:
I think the best would be to filter the attributes and only propagate `cfg(something)` for now.